src/helpers.php

```diff
@@ -18,3 +18,22 @@ function getModelForGuard(string $guard)
             })->get($guard);
     }
 }
+
+if (! function_exists('currentAuthGuard')) {
+    /**
+     *
+     * @return string|null
+     */
+    function currentAuthGuard()
```

Comment:
Can this be done without adding a global scope helper function?

Reply:
A reasonable place could be `Spatie\Permission\Guard` as a static method.
Don't know if that is a good place to have a dependancy to the `Auth` Facade/Class.